lighthouse-core/test/scripts/lantern/constants-test.js

```diff
@@ -0,0 +1,188 @@
+/**
```

Comment:
maybe add this new test to tsconfig whitelist?

Reply:
done!